src/openms/source/FORMAT/FeatureXMLFile.cpp

```diff
@@ -488,6 +491,7 @@ namespace OpenMS
       {
         map_->setUniqueId(unique_id);
       }
+      last_meta_ = map_;
```

Comment:
hmm, as far as I understand, this guarantees that any meta values that we read in after this get attached to the map itself, correct?